_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+States and [GDPR](http://data.europa.eu/eli/reg/2016/679/oj) in the
+European Union) and used profitably by the corporation.
+
+Both of these disciplines have much in common and often overlap in
+terms and structure.  In terms of IrishGen, Digital Curation is
+probably more important but both disciplines emphasise having data
+policies which guide the collection and organisation of data
+artefacts.  As stated by Eystein Thanisch, the primary policy of
+IrishGen is to take the original data seriously, even if that means,
+when working with digital editions like those on
+[CELT](https://celt.ucc.ie/), going back to the original manuscript
+source to check on possible transcription errors originating from the
+translation of the information from manuscript to printed source to
+digital source.
+
+Unfortunately, due to lack of resources and lack of time on the
+current curator's part, the insights from these two disciplines have
+not been explored in detail.  When creating databases like IrishGen,
+having a curator or a willing volunteer from one of these disciplines
+can be a boon to disrupting patterns of error before they reach a
+point at which an end user, such as a scholar or other interested party,
+encounters the data.
+
+## Automation
+
+Unlike the scribes and redactors of early medieval Ireland, who mainly
+relied on repetition and structure to reduce errors, IrishGen relies
+on those to create automated tools to translate digital versions of
+the genealogies, most prominently the CELT digital transcriptions,
+into [RDF](http://www.w3.org/TR/rdf11-primer/).
+
+However, automation is no panacea as even slight variations source
+material can cause the automation to introduce its own errors.  In
+IrishGen, a balance is struck between full automation and full manual
+with special case utilities that take some of the manual work out but
+leaving the integration to the curator.  In this case, there are two
+cases which are particularly easy for a computer to handle.  First, is
+the "String Pedigrees" ([Matthew
+Holmberg](http://isni.org/isni/0000000419902696), '[Towards a Relative
+Chronology of The Milesian Genealogical
+Scheme](https://dash.harvard.edu/handle/1/37945002)', pg. 18, for
+instance from LL's [Genelach Dal Copri
+Arad](https://celt.ucc.ie/published/G800011F/text002.html):
+
+```
+Flaithbertach m Crunmael m Commáin m Fhinain m Faigfhir m Ernine m Feic m. Ieir m Gussa m Fobrich m Maeil m. Ainmerech m Fir Roith m Muine m Fir Nued m Fir Lugdach m Buain m Argatibair m Corpri Cluchechair m Con Corbb.
+```
+
+These kinds of String Pedigrees are easy to automate and IrishGen has
+a tool to do some of this.  It is up to the curator to integrate the
+output into the file that they are working on and find any links
+between the individuals in the String Pedigree and other parts of the
+database.  This integration process is human intervention and can
+cause errors to creep in.
+
+The second point of automation is the numerical pedigrees, for 
```

Comment:
I'm not sure we can call these "pedigrees". A pedigree proves something about someone (usually nobility) by setting out their descent from an appropriate person, whereas this merely shows someone's parentage. "numerical components of pedigrees"?